app/models/like.rb

```diff
@@ -0,0 +1,13 @@
+class Like < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  after_save :update_like_counter
+
+  private
+
+  def update_likes_counter
+    post.likes_counter += 1
```

Comment:
- [ ] Line no. `10` is not necessary, As the `increment!` method is doing the same work. You can remove that line.